src/components/CartPage/CartListItem.tsx

```diff
@@ -0,0 +1,131 @@
+import styled from 'styled-components';
+
+import QuantityCounter from '../common/QuantityCounter';
+import CheckIconImage from '../../asset/check_icon.svg';
+import useCount from '../../hooks/useCount';
+import { Product } from '../../type/product';
+import DeleteButton from './DeleteButton';
+import { useEffect, useRef, useState } from 'react';
+import { useRecoilState, useSetRecoilState } from 'recoil';
+import { cartSelects } from '../../atoms/cartSelects';
+import { cartRequestAction } from '../../atoms/cartState';
+
+interface CartListItemProps {
+  id: number;
+  quantity: number;
+  product: Product;
+}
+
+export default function CartListItem({
+  id,
+  quantity,
+  product,
+}: CartListItemProps) {
+  const { count, setCount } = useCount(quantity);
+  const { name, imageUrl, price } = product;
+  const [check, setCheck] = useState(false);
+  const [cartSelectsState, setCartSelectsState] = useRecoilState(cartSelects);
```

Comment:
cartSelects에 대한 리코일 상태에 대한 이름을 어떻게 지어야할지 고민이에요. State라는 말이 무언가 불용어처럼 보여서요..!

Reply:
```suggestion
  const [cartSelects, setCartSelects] = useRecoilState(cartSelectsState);
```
와 같은 방식은 어떨까요? 사실 사용하는 측에서는 cartSelects가 props로 받아왔는지, 리코일에서 관리하고 있는 상태인지를 꼭 알 필요는 없을 것 같아요. 윤생이 이를 별도로 네이밍하고자 하는 이유가 있을까요?